content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+that is listed on the *Manage Jenkins >> Configuration as Code* UI page.
+By default, this is _$JENKINS_HOME/jenkins.yaml_.
+
+For a simple exercise to work through the process,
+you can modify the value of the "System Message"
+that is displayed on the Jenkins dashboard.
+
+. Open the JCasC YAML file with the text editor of your choice.
+. Find the `systemMessage` line near the top of the file:
++
+[source,yaml]
+----
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code plugin\n\n"
+----
+
+. Modify the text between the quotation marks to contain your new text
+. Write/save the file
+. Click the *Reload existing configuration* button to apply the changes
+. View the modified "System Message" on your dashboard
+
+[NOTE]
+====
+It is not necessary to restart Jenkins to apply the JCasC changes,
+although you should try to restart Jenkins with the modifications
+before you check the modified YAML file into SCM,
+especially when making more substantive configuration changes.
+====
+
+When you have made and tested your desired changes,
+push the modified JCasC YAML file to your SCM.
+
+== Configuring a Plugin with JCasC
+
+To configure a plugin with JCasC:
+
+. Use the UI of the current system to install and configure the plugin
+. Click *Apply >> Save* to save the configuration
+. Use *Manage Jenkins >> Configuration as Code >> View Configuration*
+to view the JCasC file with the plugin configured
+. Click on *Download Configuration*  to save the modified configuration file locally
+. Edit the JCasC YAML file to modify the configuration, if necessary
+. Save the file
+. Click *Reload existing configuration* to load the local changes onto the Jenkins server
+. Verify the changes on the UI
+. When you have thoroughly tested the plugin configuration,
+push the modified YAML file to your SCM
+
+See the
+link:https://www.jenkins.io/blog/2021/05/20/configure-plugins-with-jcasc/[Configure Plugins with JCasC]
+blog for detailed instructions
+and an embedded video demonstration of this process.
+
+== YAML File Location
+
+By default, the YAML file for the CasC configuration
+is located in `$JENKINS_HOME/jenkins.yaml`.
+The location and name of the file being used is displayed
+on the *Configuration as Code* UI page.
+You can specify a different file to view by typing the full pathname
+into the *Path or URL* field.
+
+You can specify a different location or a different file name
+for the creation of the JCasC YAML file by doing either of the following:
+
+* Populate the `CASC_JENKINS_CONFIG` environment variable to point to
+a comma-separated list that defines where configuration files are located.
+
+* Use the `casc.jenkins.config` Java property to control the file name and location.
+This is useful when installing Jenkins via a package management tool.
+It can not set an environment variable outside of a package-managed file,
+because it could be overwritten by an update.
```

Comment:
I'm not clear on this phrasing.  Is this guiding the reader to modify files that are not controlled by the package manager so that later updates by the package manager won't overwrite the files?  If that's the case, could it be rephrased as
```suggestion
Most package management systems support configuration files that will be retained across upgrades.
It is best to not modify a file installed by a package manager
because it could be overwritten by an update.
```